NBitcoin/RPC/RPCClient.Wallet.cs

```diff
@@ -91,40 +93,281 @@ wallet			 getbalance
+
+		void UnlockUnspent(params OutPoint[] outpoints);
+
+		Task UnlockUnspentAsync(params OutPoint[] outpoints);
+
+		Transaction SignRawTransaction(Transaction tx);
+
+		Task<Transaction> SignRawTransactionAsync(Transaction tx);
+
+		SignRawTransactionResponse SignRawTransactionWithWallet(SignRawTransactionRequest request);
+
+		Task<SignRawTransactionResponse> SignRawTransactionWithWalletAsync(SignRawTransactionRequest request);
+
+		WalletProcessPSBTResponse WalletProcessPSBT(PSBT psbt, bool sign = true, SigHash hashType = SigHash.All, bool bip32derivs = false);
+		Task<WalletProcessPSBTResponse> WalletProcessPSBTAsync(PSBT psbt, bool sign = true, SigHash sighashType = SigHash.All, bool bip32derivs = false);
+		WalletCreateFundedPSBTResponse WalletCreateFundedPSBT(
+			TxIn[] inputs,
+			Tuple<Dictionary<BitcoinAddress, Money>, Dictionary<string, string>> outputs,
+			LockTime locktime,
+			FundRawTransactionOptions options = null,
+			bool bip32derivs = false
+			);
+
+		Task<WalletCreateFundedPSBTResponse> WalletCreateFundedPSBTAsync(
+		TxIn[] inputs,
+		Tuple<Dictionary<BitcoinAddress, Money>, Dictionary<string, string>> outputs,
+		LockTime locktime = default(LockTime),
+		FundRawTransactionOptions options = null,
+		bool bip32derivs = false
+		);
+
+		WalletCreateFundedPSBTResponse WalletCreateFundedPSBT(
+			TxIn[] inputs,
+			Dictionary<BitcoinAddress, Money> outputs,
+			LockTime locktime,
+			FundRawTransactionOptions options = null,
+			bool bip32derivs = false
+		);
+
+		WalletCreateFundedPSBTResponse WalletCreateFundedPSBT(
+			TxIn[] inputs,
+			Dictionary<string, string> outputs,
+			LockTime locktime,
+			FundRawTransactionOptions options = null,
+			bool bip32derivs = false
+		);
+
+		Task<RPCResponse> SendCommandAsync(RPCOperations commandName, params object[] parameters);
+
+	}
+
+	public partial class RPCClient : IRPCWalletClient
+	{
+		public RPCClient GetWallet(string walletName)
+		{
+			RPCCredentialString credentialString;;
+
+			if (_BatchedRequests is null)
+			{
+				credentialString = RPCCredentialString.Parse(CredentialString.ToString());
+			}
+			else
+			{
+				if (string.IsNullOrEmpty(CredentialString.WalletName))
+				{
+					credentialString = CredentialString;
+				}
+				else
+				{
+					throw new InvalidOperationException("Batch RPC client already has a wallet assigned.");
+				}
+			}
+			credentialString.WalletName = walletName;
+
+			return new RPCClient(credentialString, Address, Network)
+			{
+				_BatchedRequests = _BatchedRequests,
+				Capabilities = Capabilities,
+				RequestTimeout = RequestTimeout,
+				_HttpClient = _HttpClient,
+				AllowBatchFallback = AllowBatchFallback
+			};
+		}
+
+		public async Task<IRPCWalletClient> CreateWalletAsync(string walletNameOrPath, 
+			bool disablePrivateKeys = false, 
+			bool blank = false, 
+			string passphrase = null, 
+			bool avoidReuse = false,
+			bool descriptors = false,
+			bool? loadOnStartup = null)
```

Comment:
please use a structure for that with proper nullables.

The defaults change from one release to the other, so we should not try to guess them.